src/V3Global.h

```diff
@@ -141,6 +143,11 @@ class V3Global final {
 
     // ACCESSORS (general)
     AstNetlist* rootp() const VL_MT_SAFE { return m_rootp; }
+    V3ThreadPool* threadPoolp() const VL_PURE { return m_threadPoolp; }
+    void threadPoolp(V3ThreadPool* threadPoolp) {
+        UASSERT(!m_threadPoolp, "call once");
```

Comment:
```suggestion
        UASSERT(!m_threadPoolp, "attempted to create multiple threadPool singletons");
```

(If a user ever reports the error we want the message distinctive.)